src/main/webapp/WEB-INF/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Air Quality Map</title>
+    <link rel="stylesheet" href="style.css">
+    <!--
+    <style>
```

Comment:
I actually deleted this commented out code a while back because I know we shouldnt have commented code that wont be used (as discussed before). Line 7 and 8 isnt on my index.html and I havent touched it since i PRed. this is weird, let me double check this

Reply:
@sbland-g  hey sai, this index.html and css within the WEB-INF folder is actually deleted code from the directory. I didn't push these files in my latest PR so these arent the file to be reviewed (they should have been deleted)

Reply:
@sbland-g  ah! i figured it out. i deleted these files a while back as they were in the wrong location. however they were in my first commit to the repo, and i had to pull from master to get rachels commits before i pushed mine on, which pulled  these index.html and css file back into my WEB-INF folder.